Scripts/fastlane/Deliverfile

```diff
@@ -7,12 +7,6 @@ app_identifier "org.wordpress"
 # Make sure to update these keys for a new version
 app_version "16.5"
 
-privacy_url({
-  'default' => 'https://automattic.com/privacy/',
-})
-
-copyright('2021 Automattic Inc.')
```

Comment:
So does that mean you confirm we can remove that from our `Deliverfile` in all our repos now that we have the `copyright.txt` in `metadata`? That's great, avoids having 2 sources of truth! 🎉

Reply:
Yep that's right – we should move this to `.txt` files instead of the `Deliverfile` in all repos (which we're now doing in WCiOS too!)